src/commands/build.yml

```diff
@@ -84,6 +100,15 @@ steps:
             dockerfile: <<parameters.path>>/<<parameters.dockerfile>>
             debug: <<parameters.debug>>
 
+  - when:
+      condition: <<parameters.use-workspace>>
+      parameters:
```

Comment:
No parameters should be defined here. Parameters are defined within the top-level parameters key for the command.